lldb/source/Expression/Materializer.cpp

```diff
@@ -540,9 +537,9 @@ class EntityVariableBase : public Materializer::Entity {
           return;
         }
 
-        if (data.GetByteSize() <
+        if (data_or_err->GetByteSize() <
```

Comment:
I like to add a 
```
auto data = *data_or_err;
```
than there is no question about whether we forgot to check the error when accessing data. Makes to code easier to read.